deps/github.com/openshift/kubernetes/REBASE.openshift.md

```
@@ -512,26 +556,24 @@ etcd version 3.5.6 or greater required
 
 Here are the steps:
-1. Create a new BugZilla with the respective OpenShift version to rebase (Target Release stays ---),
-   Prio&Severity to High with a proper description of the change logs.
-   See [BZ2021468](https://bugzilla.redhat.com/show_bug.cgi?id=2021468) as an example.
+1. Create a new Jira ticket under OCPBUGS with the respective OpenShift version to rebase,
+   Target Backport Version to the previous minor version + .z
 2. It's best to start off with a fresh fork of [openshift/kubernetes](https://github.com/openshift/kubernetes/). Stay on the master branch.
-3. This script requires `jq`, `git`, `podman` and `bash`, `gh` is optional.
+3. This script requires `git`, `podman` and `bash`, and optionally uses `gh` to create a pull request.
 4. In the root dir of that fork run:
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)